private/buf/cmd/buf/buf.go

```diff
@@ -146,6 +146,14 @@ func NewRootCommand(name string) *appcmd.Command {
 					stats.NewCommand("stats", builder),
 					migratev1beta1.NewCommand("migrate-v1beta1", builder),
 					studioagent.NewCommand("studio-agent", noTimeoutBuilder),
+					{
```

Comment:
Yea I think id still argue to put this under registry...other commands may be bsr-API-specific, but we call the registry command the subcommand to manage BSR assets

Reply:
Ye that's fair, we'll first need to remove the existing `buf beta registry plugin` (now deprecated) set of commands to avoid confusion.

```sh
Available Commands:
  create      Create a Protobuf plugin
  delete      Delete a Protobuf plugin
  list        List plugins on the specified BSR
  version     Manage Protobuf plugin versions
```

Reply:
You can do this in the same PR - `beta` commands are inherently unstable

Reply:
Done. https://github.com/bufbuild/buf/pull/2137/commits/8622c068ca7dacd2beae229d2c1eddd8c11ba610